app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 # Top-level documentation comment for User module
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly include the `:confirmable` module of `devise` in this file. It will enable users to confirm their email after signup actions. Remember that one of the requirements is to activate the `:confirmable` feature of `devise.` Kindly implement it. 👍 💻

Reply:
@Gambit142 ,

When I include the `:confirmable`, the login doesn't take me to the `user#index` page. Please, how do I go about it?

Reply:
This change has been added well